events/index.html

```diff
@@ -9,6 +9,12 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> 27-29 January, 2021</h6>
+                <h2><a href="workshops-gbi-eosc-january-2021.html">OME Workshops, virtual course</a></h2>
```

Comment:
Thoughts on adding "GBI/EOSC" here?

Reply:
I am all for it, maybe

``OME workshops run within a GBI.EOSC-Life course``

? cc @jburel

Reply:
make sense